### PR TITLE
W-15254663 enable disable user invites

### PR DESCRIPTION
**Pull request by sarathecoubian**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/external-identity-index.adoc [hunk 1]
```diff
@@ -70,3 +70,20 @@ include::partial$include-nav-identity-providers.adoc[]
...
+To modify account creation parameters:
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+include::partial$include-nav-identity-providers.adoc[]
+
+. Click *Anypoint*.
+. Select one of the following:
+.. *Allow account creation*: Organization administrators can invite new users using the *Users* tab. Accounts created using this feature belong to the Anypoint identity provider and log in using their credentials rather than SSO.
+.. *Disable account creation*: Accounts must join the organization using an external identity provider. Existing accounts are unaffected and can continue to log in using their credentials.
```
Comment (lshanmugavel) [inline modules/ROOT/pages/external-identity-index.adoc]: @sarathecoubian Maybe we could rephrase this to explicitly state that the changes are applicable to the new accounts, wdyt?
Something like
`New accounts must join the organization using an external identity provider and log in via SSO. Existing accounts are unaffected and can continue to log in using their credentials.`

Comment (sarathecoubian) [reply modules/ROOT/pages/external-identity-index.adoc]:
```suggestion
.. *Disable account creation*: New accounts must join the organization using an external identity provider and log in using SSO. Existing accounts are unaffected and can continue to log in using their credentials.
```

**Review by tasmoore — APPROVED** [review]
